src/payment.ts

```diff
@@ -37,20 +37,23 @@ export class Payment {
 
   public async createPixPaymentRequest(
     pixKey: PixKey,
-    amount: number
+    amount: number,
+    message?: string
   ): Promise<PixPaymentRequest> {
     const savingsAccountId: string = await this._context.account.getId();
 
     const createPaymentRequestInput = {
       amount,
       pixAlias: pixKey.value,
+      message: message || null,
```

Comment:
Are you sure you need to pass `null` specifically in case you don't want to write a message? If you don't pass anything, that will be `undefined`, and on the query means "field value not specified."